codecov.yml

```diff
@@ -1,12 +1,12 @@
 coverage:
   round: down
   precision: 2
-  range: '0...100'
+  range: '90...100'
   status:
     project:
       default:
         # basic
-        target: 0%
+        target: 00%
```

Comment:
Should this be 90? That's the target we're aiming for.

Reply:
Yes, that's a typo 😅